pallets/multisig/src/lib.rs

```diff
@@ -884,7 +884,12 @@ impl<T: Config> Module<T> {
                 }
                 Err(e) => {
                     update_proposal_status(ProposalStatus::ExecutionFailed);
-                    Self::deposit_event(RawEvent::ProposalExecutionFailed(e.error));
+                    Self::deposit_event(RawEvent::ProposalFailedToExecute(
```

Comment:
I think we should still emit the old event, otherwise we will need to force an upgrade to SubQuery.